scripts/local-clusters/configs/secrets.yaml

```diff
@@ -2,3 +2,8 @@ objectStorage:
   s3:
     accessKey: minioaccesskey
     secretKey: miniosecretkey
+alerts:
+  opsGenie:
+    apiKey: not-used
+  slack:
+    apiUrl: not-used
```

Comment:
Also looks unrelated

Reply:
Or was this enough that the `yq.set` were not needed? 64e5c5d73b321c5f73d70e11fd47eba340cbc1e9 made it seem so